dotnet-trace/test.json

```diff
@@ -2,8 +2,8 @@
 	"name": "dotnet-trace",
 	"enabled": true,
 	"requireSdk": true,
-	"version": "6.0",
-	"versionSpecific": false,
+	"version": "8.0",
+	"versionSpecific": true,
```

Comment:
You should keep `versionSpecific` set to `false` so the test also runs on versions higher than .NET 8.

Reply:
Noted.

Reply:
You can read the documentation for these properties on the README at https://github.com/redhat-developer/dotnet-bunny.